### PR TITLE
precourse

### PR DESCRIPTION
**Pull request by codingwithdani**

### CODE REVIEW

File: staff/daniela/Precurso/Airlines/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    })
+    
+    
+    console.log('El coste medio de los vuelos ' + Math.trunc(totalCostFlights / flights.length) + '€')
+    
+    console.log('Hay ' + flightsWithScale + ' vuelos con escalas')
+}
+  
+skylabAirlinesFlights(flights)
+
```
Comment (Markuson) [inline staff/daniela/Precurso/Airlines/index.js]: faltan ";" en varios sitios

File: staff/daniela/Precurso/Bingo/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        if (!doYouWantToPlay){
+            return 'Gracias por jugar, ' + userName
+        }
+    }    
+}
+bingo(userCard)
+
+
+
+
```
Comment (Markuson) [inline staff/daniela/Precurso/Bingo/index.js]: faltan varios ";" en el codigo

File: staff/daniela/Precurso/Calculator-2/index.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        raizCuadrada = Math.sqrt(firstNum);
+        return 'La raiz cuadrada de ese número es ' + raizCuadrada.toFixed(3)
+    }
+  
+    if (!isNaN(secondNum)){
+        raizCuadrada = Math.sqrt(secondNum);
+        return 'La raiz cuadrada de ese número es ' + raizCuadrada.toFixed(3)
+    } 
+}
+
```
Comment (Markuson) [inline staff/daniela/Precurso/Calculator-2/index.js]: faltan ";" en varios sitios

File: staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    if (nextQuestion){
+            actualIndex = pasapalabrasGame.findIndex(letterInfo => letterInfo.letter === nextQuestion.letter)
+            console.log(actualIndex)
+           return nextQuestion.question
+    } else {
+        actualIndex = pasapalabrasGame.findIndex(letterInfo => letterInfo.status === 0)
+        nextQuestion = pasapalabrasGame[actualIndex].question
+            return nextQuestion
+    }
+}
```
Comment (paulusrex) [inline staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js]: Es redundante que lo hagas así
Quizás mejor

```
let next = listOfQuestion.findIndex((letterInfo, index) => letterInfo.status === 0 && index >= actualIndex));
if (next === -1) {
next = listOfQuestion.findIndex((letterInfo, index) => letterInfo.status === 0);
}
actualIndex = next;
nextQuestion = pasapalabrasGame[actualIndex].question;
return nextQuestion;
```

File: staff/daniela/Precurso/Calculator-2/calculator.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset='UTF-8'>
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200,400,700" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="./calculator.css" />
+    <title></title>
```
Comment (edfrias) [inline staff/daniela/Precurso/Calculator-2/calculator.html]: @codingwithdani you left a title empty here :3

File: staff/daniela/Precurso/Calculator-2/index.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset='UTF-8'>
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200,400,700" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="./calculator.css" />
+    <title></title>
```
Comment (edfrias) [inline staff/daniela/Precurso/Calculator-2/index.html]: @codingwithdani here too, empty title!

File: staff/daniela/Precurso/Calculator-2/index.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+    const sendButton = document.querySelector(".sendButton");
+    const firstNumber = document.querySelector("#firstParameter");
+    const secondNumber = document.querySelector("#secondParameter");
+    const mathResults = document.querySelector(".instructions");
+    //console.log(sendButton);
```
Comment (edfrias) [inline staff/daniela/Precurso/Calculator-2/index.js]: @codingwithdani commented code here do nothing, you can erase it

File: staff/daniela/Precurso/Pasapalabra/index.js [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+let playersRanking = [];
+
+function pasapalabras (pasapalabrasGame){
+    let userName = getName()
+    let keepPlaying = true;
+    let rightAnswers = 0;
+    let wrongAnswers = 0;
+    while (keepPlaying){
+        for (let question in pasapalabrasGame){
+            //console.log('Con la letra ' + "'" + pasapalabrasGame[question].letter + "'" + ' la pregunta es:')
```
Comment (edfrias) [inline staff/daniela/Precurso/Pasapalabra/index.js]: @codingwithdani same here, this commented line is useless

File: staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.html [hunk 9]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset='UTF-8'>
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200,400,700" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="./pasapalabra-2.css" />
+    <title></title>
```
Comment (edfrias) [inline staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.html]: @codingwithdani another empty title

File: staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+        updateLetters()
+        $btnAnswer.style.display = "none"
+        $btnPasapalabra.style.display = "none"
+        $userAnswer.style.display = "none"
+        
+        //sort del ranking
+        playersResults.push({player: userName, score: numOfRightAnswer}) 
+        rankingPlayers = sortRanking(playersResults)
+
+        $questions.innerHTML =  userName + ", has llegado al final del juego. <br><br>" + "<strong>Ranking de jugadores:</strong><br>" + rankingPlayers + "<br>"
```
Comment (edfrias) [inline staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js]: @codingwithdani double ```<br>```'s kill a lot of kitties :( 
You can make a title with a  ```<p>```  and then place the ranking into an  ```<ul>```  with your desired markup

File: staff/daniela/Precurso/Airlines/index.js (hunk 1 above)
Comment (codingwithdani) [reply staff/daniela/Precurso/Airlines/index.js]: @Markuson done! :)

File: staff/daniela/Precurso/Calculator-2/index.js (hunk 3 above)
Comment (codingwithdani) [reply staff/daniela/Precurso/Calculator-2/index.js]: Done!

File: staff/daniela/Precurso/Bingo/index.js (hunk 2 above)
Comment (codingwithdani) [reply staff/daniela/Precurso/Bingo/index.js]: Done!

File: staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js (hunk 4 above)
Comment (codingwithdani) [reply staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js]: Done! Gracias, Pablo! @paulusrex

File: staff/daniela/Precurso/Calculator-2/index.html (hunk 6 above)
Comment (codingwithdani) [reply staff/daniela/Precurso/Calculator-2/index.html]: Done!

File: staff/daniela/Precurso/Calculator-2/index.js (hunk 7 above)
Comment (codingwithdani) [reply staff/daniela/Precurso/Calculator-2/index.js]: done!

File: staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.html (hunk 9 above)
Comment (codingwithdani) [reply staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.html]: done!

File: staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js (hunk 10 above)
Comment (codingwithdani) [reply staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js]: @edfrias, done! Creo que se salvaron los gatos 🐱 thanks!

**Comment by paulusrex** [comment]
```diff
diff --git a/staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js b/staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js
index 5662431..34acd44 100644
--- a/staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js	
+++ b/staff/daniela/Precurso/Pasapalabras 2 -Proyecto final/pasapalabra-2.js	
@@ -99,7 +99,7 @@ $btnUserName.addEventListener("click", function(){
 
 function updateLetters(){
     $letters.innerHTML = ""
-    pasapalabrasGame.forEach(function(letterInfo){
+    pasapalabrasGame.forEach((letterInfo) => {
         let className = "letter"
         console.log(letterInfo.status)
         if (letterInfo.status === STATUS_OK){

```

**Pull request merged**